### PR TITLE
Updated German Language Files

### PR DESCRIPTION
**Pull request by ErrorCraftLP**

### CODE REVIEW

File: src/main/resources/assets/galacticraftcore/lang/de_DE.lang [hunk 1]
```diff
@@ -352,23 +352,23 @@ gui.status.on.name=AN
...
-gui.button.loadfuel.name=Lade Treibstoff
+gui.button.loadfuel.name=Treibstoff laden
 gui.button.refine.name=Raffiniere
 gui.button.back.name=Zurück
 gui.button.next.name=Weiter
 gui.button.unlockschematic.name=Bauplan freischalten
-gui.button.stoprefine.name=Stoppe Raffinieren
-gui.button.enable.name=Aktiviere
-gui.button.disable.name=Deaktiviere
+gui.button.stoprefine.name=Raffinieren stoppen
```
Comment (XFactHD) [inline src/main/resources/assets/galacticraftcore/lang/de_DE.lang]: I think "Raffination stoppen" sounds better.

File: src/main/resources/assets/galacticraftcore/lang/de_DE.lang [hunk 2]
```diff
@@ -414,11 +414,12 @@ gui.rocket.inv.name=Inventar / Treibstoff
 gui.rocket.warning.nogyroscope=Mysteriöse Kräfte sind im Spiel, die Gyroskope spielen verrückt!
 gui.rocket.warning.noparachute=Ich habe keinen Fallschirm! Wenn ich noch einmal Starten drücke, gibt es kein Zurück!
 gui.rocket.warning.nofuel=Ich brauche erst einmal etwas Raketentreibstoff!
-gui.chest.warning.wrongkey=Ich brauche wohl einen Dungeonschlüssel Stufe %s, um das zu entsperren!
+gui.chest.warning.wrongkey=Ich brauche wohl einen Dungeonschlüssel der Stufe %s, um das zu entsperren!
+gui.bedFail.message=Ich brauche eine kryogene Kammer, wenn ich im Weltraum schlafen möchte!
 gui.frequencymodule.warning0=Ich brauche wohl ein
```
Comment (XFactHD) [inline src/main/resources/assets/galacticraftcore/lang/de_DE.lang]: "Frequenzmodul" is missing here.

Comment (ErrorCraftLP) [inline src/main/resources/assets/galacticraftcore/lang/de_DE.lang]: I looked at the English language file and it's missing there, too. Maybe it gets inserted later?

Comment (XFactHD) [inline src/main/resources/assets/galacticraftcore/lang/de_DE.lang]: @radfast any idea why this is also missing in the English language file?

**Comment by radfast** [comment]
@XFactHD is it working in German?  If so, don't complain...
I think maybe the code writes "Ich brauche wohl ein" then copies the name "Frequenzmodul" from the item name.  Maybe even in a different colour?

**Comment by radfast** [comment]
Thank you @ErrorCraftLP for the update.

**Comment by ErrorCraftLP** [comment]
@radfast was right. I found this code in the GCPlayerHandler class:
```java
player.addChatMessage(new ChatComponentText(EnumColor.YELLOW + GCCoreUtil.translate("gui.frequencymodule.warning0") + " " + EnumColor.AQUA + GCItems.basicItem.getItemStackDisplayName(new ItemStack(GCItems.basicItem, 1, 19)) + sb.toString()));
```

This code takes the first part of message and adds the name of the frequency module to it (with a different color).

**Comment by XFactHD** [comment]
Ok, didn't know that and couldn't check because I am currently not using Galacticraft.

**Pull request merged**